react/components/LocaleSwitcher/GlobeIcon.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+const GlobeIcon = ({
```

Comment:
We need this icon in `store-icons`

Reply:
done in this PR: https://github.com/vtex-apps/store-icons/pull/27/